prow/config.yaml

```diff
@@ -32,7 +32,7 @@ plank:
       gcs_configuration:
         bucket: "knative-prow"
         path_strategy: "explicit"
-      gcs_credentials_secret: "test-account"
+      gcs_credentials_secret: "gcs-upload"
```

Comment:
IIRC Prow supports configuring `default_decoration_configs` per build cluster, AKA replacing `*` to be build cluster names, so you won't have to override this per job.

You can check the Prow docs or config spec to check whether Prow supports this and/or how to configure it.

Reply:
I didn't know that. That would my life so much easier

Reply:
Prow does support this - see an example at https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/oss/config.yaml#L93-L175

Reply:
Done. Change is much leaner now.